BaseClasses.py

```diff
@@ -566,9 +566,10 @@ def location_condition(location: Location):
 
         def location_relevant(location: Location):
             """Determine if this location is relevant to sweep."""
-            if location.progress_type != LocationProgressType.EXCLUDED \
-                and (location.player in players["locations"] or location.event
-                     or (location.item and location.item.advancement)):
+            if __debug__ \
```

Comment:
The accessibility check is currently skipped if players are on items, which is the default and what *most* users actually experience currently. Since the option is being removed, this gives players essentially the same behavior as before, but enforces proper accessibility during development.

Reply:
That's not quite true; even on items the location is checked if its item is advancement. At least it was, now the safety net is gone, which feels wrong.

Reply:
removed the `__debug__` completely but i'm still unsure how this should be handled. it feels wrong to force every world that doesn't define the optional accessibility to have every single one of their locations checked after fill, especially since it isn't optional. i do agree now that was probably the wrong way to do it but there should still probably be some way to disable it or at least only use the safety net.